pkg/api/universal/secrets.go

```diff
@@ -17,9 +17,10 @@ import (
 	"context"
 	"time"
 
+	apierrors "github.com/dapr/dapr/pkg/api/errors"
```

Comment:
can you move this down into the section with all the other `"github.com/dapr/dapr...` imports?